wgengine/magicsock/magicsock.go

```diff
@@ -3544,13 +3544,16 @@ func (de *endpoint) removeSentPingLocked(txid stun.TxID, sp sentPing) {
 	delete(de.sentPing, txid)
 }
 
-// sendDiscoPing sends a ping with the provided txid to ep.
+// sendDiscoPing sends a ping with the provided txid to ep using discoKey.
```

Comment:
clarify whose discoKey this is. ours vs ep's.

Reply:
Done.